s3_website.gemspec

```diff
@@ -28,7 +28,6 @@ Gem::Specification.new do |s|
   s.files         = `git ls-files`
                       .split("\n")
                       .reject { |f| f.match('sbt-launch.jar') } # Reject the SBT jar, as it is a big file
-                      .push('resources/s3_website.jar.md5') # Include the checksum file in the gem
```

Comment:
Thanks for the pull request, @pjkelly! One question: why did you remove this line?